engine/lite/components/kafka/src/main/scala/pl/touk/nussknacker/engine/lite/components/LiteKafkaSourceImplFactory.scala

```diff
@@ -50,4 +51,11 @@ class LiteKafkaSourceImpl[K, V](contextInitializer: ContextInitializer[ConsumerR
     initializerFun(deserialized)
   }
 
+  override def generateTestData(size: Int): Array[Byte] = formatter.generateTestData(topics, size, kafkaConfig)
```

Comment:
Don't like that we still have it in source implementation instead of factory :-( Because of this I can't see what is the real difference in flink and lite implementations

Reply:
I added trait for common `generateTestData` and added comments on testDataParser implementations